### PR TITLE
Created functions to make longitude cyclic and normalize latitude

### PR DESCRIPTION
**Pull request by kjmayer**

### CODE REVIEW

File: config/hyper_kjm.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    trainer:training_loss:
+      type: "categorical"
+      settings:
+        name: "training_loss"
+        choices: ["mae", "mse", "smooth", "logcosh", "xtanh", "xsigmoid", "huber"]
+    model:name:
+      type: "categorical"
+      settings:
+        name: "model_name"
+        choices: ["unet", "unet++", "manet", "linknet", "fpn", "pspnet", "pan", "deeplabv3", "deeplabv3+"]
```
Comment (mariajmolina) [inline config/hyper_kjm.yml:66]: Let's remove the "pan" model from the list of choices -- it doesn't work with our data

File: config/hyper_kjm.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    model:encoder_name:
```
Comment (mariajmolina) [inline config/hyper_kjm.yml:67]: note that these are the pre-trained weights that require nodes with internet access to download -- if using nodes without internet access, just comment this out

File: config/hyper_kjm.yml (hunk 1 above)
Comment (mariajmolina) [reply config/hyper_kjm.yml:66]: (I have it removed from my yaml file)

**Review by mariajmolina — APPROVED** [review]

File: notebooks/CreateMLCoords.ipynb [hunk 3]
```diff
@@ -1,6 +1,7 @@
 {
```
Comment (mariajmolina) [inline notebooks/CreateMLCoords.ipynb:1]: <p>why did you comment this out -- is the xesmf not in the environment?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mariajmolina/s2sml/pull/13/discussion/#comment-1223329538'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (kjmayer) [reply notebooks/CreateMLCoords.ipynb:1]: Yeah, I don't think it's in the s2sml-env, but I didn't check the s2sml-analysis environment.

**Pull request merged**